src/dcd/server/autocomplete/complete.d

```diff
@@ -47,6 +47,11 @@ import dsymbol.utils;
 import dcd.common.constants;
 import dcd.common.messages;
 
+enum CompletionToken {
+	none,
+	bracket, bang
+}
```

Comment:
I think something like this would be better here:

```suggestion
enum CallTipHint {
	none, // asserts false if passed into setCompletion with CompletionType.calltips
	regularArguments,
	templateArguments,
	indexOperator,
}
```